Makefile.am

```diff
@@ -160,14 +160,23 @@ baseline_filtered_combined.info: leveldb_baseline_filtered.info baseline_filtere
 	$(LCOV) -a leveldb_baseline_filtered.info -a baseline_filtered.info -o $@
 
 test_bitcoin.info: baseline_filtered_combined.info
-	$(MAKE) -C src/ check
+	$(MAKE) -C src/ bitcoin_test_check
```

Comment:
Does this change leave any vestigial build directives? I don't see any other accompanying changes.